From 9e88f841c8f89e0984d4c6734f0b32cd68ee011c Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <josef.schlehofer@nic.cz>
Date: Sat, 15 Jun 2019 19:39:13 +0200
Subject: [PATCH] opkg: dont use --force-overwrite

Users of advanced administration called LuCI when they want to install
package in LuCI are not aware that it calls
opkg install package --force-overwrite which will solve the coflict, if
there is any.

When some packages are in conflict then some update of Turris OS cannot
proceed properly and successfully and users in those cases don't know
that they did something wrong.

Github issue: https://github.com/openwrt/luci/issues/2771
---
 modules/luci-base/luasrc/model/ipkg.lua | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/luci-base/luasrc/model/ipkg.lua b/modules/luci-base/luasrc/model/ipkg.lua
index e27ea5289..614cdf6ed 100644
--- a/modules/luci-base/luasrc/model/ipkg.lua
+++ b/modules/luci-base/luasrc/model/ipkg.lua
@@ -12,7 +12,7 @@ local pairs = pairs
 local error = error
 local table = table
 
-local ipkg = "opkg --force-removal-of-dependent-packages --force-overwrite --nocase"
+local ipkg = "opkg --force-removal-of-dependent-packages --nocase"
 local icfg = "/etc/opkg.conf"
 
 module "luci.model.ipkg"
-- 
2.20.1

